dev_mode/webpack.prod.config.js

```diff
@@ -21,7 +21,8 @@ config[0] = merge(config[0], {
         },
         cache: process.platform !== 'win32'
       })
-    ]
+    ],
+    splitChunks: { chunks: 'all' }
```

Comment:
I assumed it was safe because we use it in the other webpack config, but I'm removing it based on the PR @blink1073 posted below and leaving just the increased memory command line argument to see if the integrity build is still happy.